examples/user/README.md

```diff
@@ -0,0 +1,30 @@
+## User Controller
+
+This is an example CompositeController that manages Cluster scoped resources.
```

Comment:
I just realized this is a CompositeController, which I think we decided would need more work before we could officially say it supports cluster-scoped parents. It's got a bigger surface area to check, like controller revisions for rolling update. I think we should hold off on merging a cluster-scoped CompositeController example until we know more about what works, since people will assume an example means it's fully supported.

In the meantime, this would probably still work as a DecoratorController, so we could still have a test for cluster-scoped parent+child. Then we can change it to a CompositeController when that's officially supported.

Reply:
I think a CompositeController that does not use the RollingUpdate feature will work but you're right, I don't think that's clear so I'll switch it to a decorator controller.